docs/user-guide/parallel-inference.md

```diff
@@ -9,7 +9,8 @@ section](#concurrency-in-python) below.
 
 ![](../assets/parallel-inference.svg)
 
-By default, MLServer will spin up a pool of 4 workers for each loaded model.
+By default, MLServer will spin up a pool of `N` worker processes to run
```

Comment:
Would the expectation not be that perhaps only 1 worker be expected instead of N?

Reply:
By default the pool size is the number of available cores. Should we lower that default?

Reply:
It was just a general - I would say 1 May be more intuitive as the default, but I don't think it should block the pr - my thinking is that for larger models like transformers it would be a surprise of the sheer amount of memory loaded by default - I think this was also the reason why in sc v1 python wrapper the default was 1 as well.

Reply:
That's a great point! Let's lower that default to 1.